app-devices/yubioath-flutter/autobuild/defines

```diff
@@ -0,0 +1,9 @@
+PKGNAME=yubioath-flutter
+PKGSEC=admin
+PKGDEP="ccid pcsclite yubikey-manager gtk-3 libappindicator libnotify"
+BUILDDEP="swig fvm uv llvm desktop-file-utils"
+PKGDES="Yubico Authenticator for Desktop"
+PKGREP="yubioath-desktop"
```

Comment:
Specify the version of `yubioath-desktop` like `PKGREP="yubioath-desktop<=5.1.0-2`. Add `PKGBREAK="yubioath-desktop<=5.1.0-2"`. You also need to make `yubioath-desktop` a dummy transitional package. You may search "transitional" in the tree to look for examples.